src/main/resources/db/migration/V4__add_division_table.sql

```diff
@@ -0,0 +1,10 @@
+create sequence division_seq start with 1 increment by 1;
+
+create table division
+(
+    id                    bigint default nextval('division_seq') primary key,
+    name                  varchar(255) unique not null,
```

Comment:
I don't think this should be a unique column? It might need a unique constraint combined with the `original_company_name` but I don't think we want the name to be unique on its own.